website_sale_wishlist_keep/README.rst

```diff
@@ -0,0 +1,84 @@
+==========================
+Website Sale Wishlist Keep
+==========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fe--commerce-lightgray.png?logo=github
+    :target: https://github.com/OCA/e-commerce/tree/12.0/website_sale_wishlist_keep
+    :alt: OCA/e-commerce
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_wishlist_keep
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/113/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module sets the checkbox, that keep the products in the wishlist, active by default.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+#. Go to shop
+#. Add 1 product to the wishlist
+#. Go to the wishlist
+#. Active Customize > Keep items on wishlist
+#. You can see the option checked allways that you go to the wishlist.
```

Comment:
```suggestion
#. You can see the "Add product to my cart but keep it in my wishlist" option checked by default
```

Reply:
Yhe first step that you suggest to delete is needed, but probably someone do it before you